src/hCassandra_test.py

```diff
@@ -0,0 +1,181 @@
+__author__ = 'annyz'
+
+import sys
+import logging
+import math
+import ast
+
+from datetime import datetime, timedelta
+from optparse import OptionParser
+from pprint import pformat  # NOQA
+from hydra.lib import util
+from hydra.lib.h_analyser import HAnalyser
+from hydra.lib.hydrabase import HydraBase
+
+try:
+    # Python 2.x
+    from ConfigParser import ConfigParser
+except ImportError:
+    # Python 3.x
+    from configparser import ConfigParser
+
+l = util.createlogger('hCassandra', logging.DEBUG)
+
+
+class RunTestCassandra(HydraBase):
+    def __init__(self, options, runtest=True, mock=False):
+        self.options = options
+        self.config = ConfigParser()
+        HydraBase.__init__(self, 'CassandraStressTest', self.options, self.config, startappserver=runtest, mock=mock)
```

Comment:
For hydra to get access to your code on slaves it needs the app path's
Add an argument to HydraBase init 
app_dirs=['src', 'hydra']
this will make the src and hydra directories available on slaves when running.